src/Chunk.cpp

```diff
@@ -1715,10 +1715,12 @@ void cChunk::SetAlwaysTicked(bool a_AlwaysTicked)
 	if (a_AlwaysTicked)
 	{
 		m_AlwaysTicked += 1;
+		Stay(a_AlwaysTicked);
```

Comment:
So is m_AlwaysTicked now equivalent to m_StayCount? Or is there something preventing chunks from being ticked? I'm presuming the latter.